### DISCUSSION TITLE
I haven't get my GitHub education benefits till now.

### DISCUSSION THREAD

**Discussion by ahmedaboseada** [community/community#145126] [GitHub Education]
### Hi everyone,

I haven't get my GitHub education benefits till now.
I requested on 17 Nov, and got approved and till this moment nothing happened
![Screenshot 2024-11-20 170117](https://github.com/user-attachments/assets/ac02a7d1-fe6d-4211-87ab-b3947f40be24)
![Screenshot 2024-11-20 170142](https://github.com/user-attachments/assets/ccb49c59-c7be-4b4f-b462-8d9deebad75d)

**Comment by davevad93** [comment]
Hi @ahmedaboseada ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ahmedaboseada was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
